src/js/components/validateData/validateValues/ValidateValuesErrorReport.jsx

```diff
@@ -66,7 +66,7 @@ export default class ValidateValuesErrorReport extends React.Component {
 
             let description = item.error_description;
             if (item.error_name == 'rule_failed') {
-                description = item.rule_failed;
+                description = ['Rule ' + item.original_label + ': ' + item.rule_failed];
```

Comment:
It seems redundant to keep `Failed rule:` in the display. If you look at the cross-file validations, it just says `Rule LABEL: DESCRIPTION`. I think that's a backend change but I think we should keep it consistent